package.json

```diff
@@ -101,6 +105,7 @@
     "rollup": "^2.28.2",
     "rollup-plugin-vue": "^5.*.*",
     "rpm-builder": "^1.2.1",
+    "run-script-os": "^1.1.1",
```

Comment:
You forgot to commit the changes to `yarn.lock` resulting from this change.

Reply:
Hm, I was using `npm`. Just pushed the lock file for both :)